hicommonwealth/commonwealth
# Remove Snapshot Controller

## Link to Issue
Closes: #8287 

## Description of Changes
The goal of this PR was to remove Snapshot Controller from the `app` object and all `app.snapshot...` occurrences from the codebase. Controller was not so big but there is a lot of logic in `snapshot_utils.ts` and `NewSnapshotProposalForm/helpers.ts` files. Because this PR is already big and long-standing, I need to stop somewhere, because the main goal is to get rid of `app` object. Overall:
- did some cleanups in directory structure and naming files (renamed from `snapshot_files` to `SnapshotFiles` convention) + moved all the scss files so they are closer to the react files
- removed snapshot controller 
- removed `apollo graphql client` and `graphql-tag` packages (this is now handled by graphql-request + react-query) 

## Test Plan
- go to community that has snapshot enabled
- make sure that Snapshots item under Governance section in left sidebar is visible
- make sure that active and ended proposals are loading
- make sure that individual proposals pages are loading along with additional informations and votes
- make sure you are able to vote/create snapshot proposal

## Deployment Plan

1. n/a

## Other Considerations

- n/a
